src/components/index/Features.tsx

```diff
@@ -0,0 +1,42 @@
+import { Card } from "./Card";
+import { SectionHeading } from "./SectionHeading";
+import { SectionLayout } from "./SectionLayout";
+
+const FEATURES_CARDS = [
```

Comment:
I think that we should create some folder called maybe `data` or something like that, where we will store all static texts and other props related to our landing page

Reply:
`/config` was created, please paste your consts there